tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "compilerOptions": {
+    "outDir": "./dist",
+    "noImplicitAny": true,
+    "target": "es5",
+    "sourceMap": true,
```

Comment:
same source map question remains here - i'm wondering if there's a way to configure this so it only generates source maps when developing and not when builds are production